MVCDemos/Views/Toolbar/Multiline.cshtml

```diff
@@ -0,0 +1,337 @@
+﻿@model DevExtreme.MVC.Demos.ViewModels.DropDownButtonViewModel
+
+@{
+    var toolbarModes = new [] { "Multiline mode", "Singleline mode" };
+    var listTypes = new [] {
+        new { icon = "orderedlist", alignment = "orderedlist", hint = "Ordered" },
+        new { icon = "bulletlist", alignment = "bulletlist", hint = "Bullet" }
```

Comment:
I think we should move all data (this and from dataSources) to a model

Reply:
Can you check this commit? 

https://github.com/DevExpress/devextreme-demos/pull/2004/commits/05c2a4c6f2f88d68152ae92b286c4eba0e1284b7